camelot/view/action_steps/gui.py

```diff
@@ -74,7 +74,8 @@ class Refresh( ActionStep, DataclassSerializable ):
     """Refresh all the open screens on the desktop, this will reload queries
     from the database"""
 
-    def gui_run( self, gui_context ):
```

Comment:
@jeffh92 : fyi : als ik 'gepruts' in de ui tegenkom ivm de geserialiseerde steps zal ik dat en passant trachten op te lossen ...